joomla/joomla-cms
# Use getState instead of direct property read

Pull Request for Issue #10416.
#### Summary of Changes

Use getState instead of direct property read in JModelList  to avoid any undefined property warnings.
Using `$this->getState` against `$this->state->get` ensures the model state is populated appropriately.
#### Testing Instructions
- Enable error reporting from global configuration.
- In any list view class such as `ContentViewArticles` at `administrator/components/com_content/views/articles/view.html.php` in the display method call `$this->get('FilterForm');` before `$this->get('State')` or `$this->get('Items')` etc.

**Expected**: Everything should work correctly.
**Actual**: You see 4 php notices such as `Notice: Undefined property: JObject::$list....`

Fixes #10416
